lib/JSON.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
At some point we should make this consistent in every file (some use !/usr/bin/env bash I believe?)